packages/roosterjs-content-model/lib/publicApi/block/setSpacing.ts

```diff
@@ -0,0 +1,11 @@
+import { IExperimentalContentModelEditor } from '../../publicTypes/IExperimentalContentModelEditor';
+import { formatParagraphWithContentModel } from '../utils/formatParagraphWithContentModel';
+
+export default function setSpacing(
+    editor: IExperimentalContentModelEditor,
+    spacing: number | string
+) {
+    formatParagraphWithContentModel(editor, 'setSpacing', paragraph => {
+        paragraph.format.lineHeight = spacing.toString();
```

Comment:
Do we need to add unit here? Such as 'px' or 'pt'?

Reply:
unitless line-height is [the preferred way](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)
we can still set px or pt if we want through spacing: string

Reply:
sounds good.